ui/app/src/models/ContentInstance.ts

```diff
@@ -22,6 +22,7 @@ export interface ContentInstanceSystemProps {
   dateCreated: string;
   dateModified: string;
   contentTypeId: string;
+  versionNumber?: string; // TODO: remove this;
```

Comment:
?

Reply:
that is because getVersions is returning a ContentInstance, but is needs versionNumber, the api is not ready, so im adding the versionNumber to the type in the meantime